.github/workflows/release.yml

```diff
@@ -18,7 +18,7 @@ jobs:
             - uses: actions/checkout@v2
             - uses: actions/setup-node@v2
               with:
-                  node-version: '16.x'
+                  node-version: '18.x'
```

Comment:
Needed, because that is the minimum supported Node.js version of `semantic-release@21.0.5`.